### DISCUSSION TITLE
My Stem on my DJ Pro is not isolating the vocals 100per cent was working before

### DISCUSSION THREAD

**Discussion by DJC95-1** [tidal-music/discussions#80] [General]
My Stem on DJ Pro not isolating my vocals 100 per cent on my Tidal Music 
Was fully working before anyone else having any issues isolating their music in Tidal

**Comment by stefanmoro** [comment]
Hey @DJC95-1 ,

This GitHub discussions forum aims at handling technical discussions around the [TIDAL Developer Platform](https://developer.tidal.com/) (TIDAL API/SDK, open source projects etc).

As this question relates to an existing product offering, I will kindly ask you to reach out to our main [TIDAL support](https://support.tidal.com/).

Thanks
